gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Wed Dec 09 14:41:33 PST 2020
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.0-all.zip
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.0-bin.zip
```

Comment:
Just reorganizing things here?

Reply:
I think Intellij did something.  I can remove it since it's not really a change. But maybe I'll leave it so IntelliJ doesn't constantly fight with us.